blazor-features/ClassStructOrRecordInCSharp/ApiData.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClassStructOrRecordInCSharp
+{
+    public record ApiData
+    {
+        public int Id { get; set; }
+    }
+
+    public record OtherApiData(string id);

```

Comment:
The naming convention for primary arguments on records is to use PascalCase as these are translated into init only properties.